src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,63 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import lotto.message.Message;
+
+public class Lotto implements Comparator<LottoNumber> {
+
+  private final List<LottoNumber> lotto = new ArrayList<>();
+
+  public Lotto(List<Integer> numberPull) {
+    checkDuplicateNumbers(numberPull);
+    creatLotteNumber(numberPull);
+  }
+
+  public Lotto() {
+  }
+
+  private void checkDuplicateNumbers(final List<Integer> numberPull) {
+    if(numberPull.size() != numberPull.stream().distinct().count()){
+      throw new IllegalArgumentException(Message.MSG_ERROR_SAME_LOTTO_NUMBER);
+    }
+  }
+
+  public List<LottoNumber> getLotto() {
+    return lotto;
+  }
+
+  private void creatLotteNumber(final List<Integer> numberPull) {
+    numberPull.forEach(number -> {
+      lotto.add(new LottoNumber(number));
+    });
+    lotto.sort(new LottoNumber());
```

Comment:
```suggestion
    lotto.sort(this::compare);
```
이렇게 코드를 수정하면 조금 더 직관적으로 코드를 이해 할 수 있지 않을까요? 🤔

Reply:
보내주신 코드 감사합니다.
해당 코드를 넣고 테스트를 수행해 본 뒤 IDE가 추천하는 코드가 있어서 추가적으로 테스트를 해보았는데요.
이상이 없어서 IDE가 추천하는 코드로 반영해보려 합니다.

numberPull.forEach(number -> lotto.add(new LottoNumber(number)));
    lotto.sort(this);